src/controller/user.controller.js

```diff
@@ -0,0 +1,95 @@
+const userService = require('../services/user.service');
+
+const get = async (req, res) => {
+  const users = await userService.getAll();
+
+  if (!users) {
+    res.send(users);
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(users);
+};
+
+const getOne = async (req, res) => {
+  const { id } = req.params;
+
+  const user = await userService.getById(+id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const update = async (req, res) => {
+  const { id } = req.params;
+  const { name } = req.body;
+  const user = await userService.getById(+id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  if (typeof name !== 'string') {
+    res.sendStatus(422);
+
+    return;
+  }
+
+  await userService.updateUser({ id, name });
+
+  const updatedUser = await userService.getById(+id);
+
+  if (!updatedUser) {
+    res.sendStatus(404);
+
+    return;
```

Comment:
Same as above, you are converting the id to a number using the unary plus operator. Better to check if the id is a number before trying to convert it.